climada/engine/impact.py

```diff
@@ -882,9 +959,16 @@ def _exp_impact(self, exp_iimp, exposures, hazard, imp_fun, insure_flag):
 
     def _build_exp(self):
         eai_exp = Exposures()
+
+        # whacky bug: geom must be set first, otherwise NaNs are introduced
```

Comment:
wording: how is it whacky?
if it's a bug - shouldn't it be fixed instead?

Reply:
I don't have the time to investigate this. It may be related to `Exposures` GDF inheritance, or it might even be a problem upstream. I could remove this, but it took me a whole while to find out, so documenting it like this seemed sensible. (And: whacky as in 'there is no discernible reason for this to happen'.)